### PR TITLE
Changelog: Fix faulty link 

### PR DESCRIPTION
**Pull request by daschuer**
This should fix a build error with the manual: 
https://github.com/mixxxdj/manual/actions/runs/3072969204/jobs/4965120078

### CODE REVIEW

**Comment by daschuer** [comment]
The annoying pre-commit issue will be solved after merging:
https://github.com/mixxxdj/mixxx/pull/10882

**Comment by daschuer** [comment]
Th Ubunutu 22.4 failure is fixed after merging https://github.com/mixxxdj/mixxx/pull/10904

File: CHANGELOG.md [hunk 1]
```diff
@@ -430,7 +430,7 @@
 * Fix crash when starting without a valid skin directory [#4575](https://github.com/mixxxdj/mixxx/pull/4575) [lp:1934560](https://bugs.launchpad.net/mixxx/+bug/1934560)
 * LateNight: Move logo to the right [#4677](https://github.com/mixxxdj/mixxx/pull/4677)
 * LateNight: Use correct tooltip for key control toggle [#4696](https://github.com/mixxxdj/mixxx/pull/4696)
-* LateNight: Add toggles to show loop and beatjump controls [#4713](https://github.com/mixxxdj/mixxx/pull/[#4713](https://github.com/mixxxdj/mixxx/pull/4713))
+* LateNight: Add toggles to show loop and beatjump controls [#4713](<https://github.com/mixxxdj/mixxx/pull/>[#4713](https://github.com/mixxxdj/mixxx/pull/4713)
```
Comment (ronso0) [inline CHANGELOG.md:433]:
```suggestion
* LateNight: Add toggles to show loop and beatjump controls [#4713](https://github.com/mixxxdj/mixxx/pull/4713)
```
and metainfo.xml will be fixed automatically

**Review by ronso0 — CHANGES_REQUESTED** [review]

**Comment by daschuer** [comment]
Done

**Comment by ronso0** [comment]
Thank you!

**Pull request merged**